src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/policies/policyV1.kt

```diff
@@ -0,0 +1,1506 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service
```

Comment:
Small point, but package could / maybe should be service.policies

Reply:
yes, totally agree. Is it alright to change in a follow up?

Reply:
No problem, nothing urgent